src/shared-components/ActionItems/ActionItemModal/ActionItemModal.autocomplete-coverage.spec.tsx

```
@@ -0,0 +1,246 @@
+            value: '',
+          },
+        })}
+      </div>
+    );
+  };
+
+  return { __esModule: true, default: Autocomplete, __autocompleteCalls };
+});
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import { __autocompleteCalls } from '@mui/material/Autocomplete';
```

ESLint

[error] '@mui/material/Autocomplete' import is restricted from being used. Do not import Autocomplete from `@mui/material`. Use the shared DropDownButton component with searchable={true} instead. If you genuinely need Autocomplete (not for a dropdown), add your file to "scripts/eslint/config/exemptions.ts".

(no-restricted-imports)